src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFHeaderLines.java

```diff
@@ -77,7 +76,7 @@ public static VCFFormatHeaderLine getEquivalentFormatHeaderLine(final String inf
         addFilterLine(new VCFFilterHeaderLine(VCFConstants.PASSES_FILTERS_v4, "Site contains at least one allele that passes filters"));
 
         addFilterLine(new VCFFilterHeaderLine(NAY_FROM_YNG, "Considered a NAY in the Yay, Nay, Grey table"));
-        addFilterLine(new VCFFilterHeaderLine(EXCESS_ALLELES,"Site has an excess of alternate alleles based on the input threshold (default is " + CommonCode.EXCESS_ALLELES_THRESHOLD + ")"));
+        addFilterLine(new VCFFilterHeaderLine(EXCESS_ALLELES,"Site has an excess of alternate alleles based on the input threshold"));
```

Comment:
Why did the info about the default go away?

Reply:
Good catch! 😄  Because this code is in a non-GVS package and it was reaching to a GVS package to grab that. There could be other, better ways of solving this problem.